community/lucene-index/src/main/java/org/neo4j/index/lucene/legacy/AbstractIndexHits.java

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.index.impl.lucene;
+package org.neo4j.index.lucene.legacy;
```

Comment:
Shouldn't we keep `impl` in there? We have quite generic fully qualified class name pattern matching where something containing `.impl.` is automatically hidden from javadocs and things like that.